### ISSUE TITLE
No checked hint of checkable actions inside menu entries

### ISSUE THREAD

**Issue by mgrojo** [ColinDuquesnoy/QDarkStyleSheet#338]
### Describe Your Environment

Please, report the output of the command (from v2.6+) or fill the information below:

`$ qdarkstyle --all`

Or use the helpdev if you have

`$ helpdev`

[Versions from your environment]

- QDarkStyle: v3.2.1
- OS: Ubuntu 22.04.3 LTS
- Python: N/A

### Language

C++

### Description / Steps to Reproduce [if necessary]

Place a checkable action in a menu. 

### Actual Result
There is no display hint about whether the menu entry is checked or not.

![image](https://github.com/ColinDuquesnoy/QDarkStyleSheet/assets/13812910/dbffcdda-ab69-48b0-8638-d3df1df9e248)

### Expected Results / Proposed Result

Clear indication of the checked entries, like in the default style:

![image](https://github.com/ColinDuquesnoy/QDarkStyleSheet/assets/13812910/6a320760-bc7e-48a8-b9c7-6e7fc245ecab)

### Relevant Code [if necessary]

Related UI definition in DB Browser for SQLite:
https://github.com/sqlitebrowser/sqlitebrowser/blob/e77ae65c11a44c00489596120aff990e4df0a9d4/src/MainWindow.ui#L823

```xml
     <string>&amp;View</string>
    </property>
    <addaction name="viewDBToolbarAction"/>
    <addaction name="viewExtraDBToolbarAction"/>
    <addaction name="viewProjectToolbarAction"/>
    <addaction name="separator"/>
   </widget>
```
```xml
  <action name="viewDBToolbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/toolbar</normaloff>:/icons/toolbar</iconset>
   </property>
```

**Comment by dedowsdi** [comment]
Meet the same problem here.

**Comment by dpizetta** [comment]
Hey @mgrojo thanks for reporting. QMenus and QComboBox are very complicated to deal with and correlated. This case is a special one where there is an icon + checkable action. You can also notice a space between the icon and name, which it was supposed to have a checkbox, or another indicator without this space as you mentioned. This is related to #308 and many other closed ones like #214. @ccordoba12 do you think @conradolandia can help us with this, since its more related to CSS?

**Comment by ccordoba12** [comment]
> @ccordoba12 do you think @conradolandia can help us with this, since its more related to CSS?

Yep, I think so. Is the idea to add a new css rule to the checked state of actions in QMenus?

**Comment by conradolandia** [comment]
I think something like this could work: 

![image](https://github.com/ColinDuquesnoy/QDarkStyleSheet/assets/5027583/7f6ea0f6-10b6-41ee-9fdc-44537e60c8ab)

**Comment by ccordoba12** [comment]
> I think something like this could work

Yep, I think so too.

> I would like to know if its possible to alter the icon in any way or if it's possible to add another icon

Probably not with css alone, but I really don't know.

**Comment by conradolandia** [comment]
> Probably not with css alone, but I really don't know.

Maybe using pseudo-classes like `.myclass:checked::after {...}`?

**Comment by dpizetta** [comment]
In the first example, maybe we can remove the extra space (where the check status go). It should be tested with checkable combo boxes too. Thanks for your attention on this.
